src/game.cpp

```diff
@@ -3978,8 +3980,8 @@ void Game::handlePointingAtObject(const PointedThing &pointed, const ItemStack &
 			if (!disable_send)
 				client->interact(0, pointed);
 		}
-	} else if (getRightClicked()) {
-		infostream << "Right-clicked object" << std::endl;
+	} else if (wasKeyDown(KeyType::PLACE)) {
+		infostream << "Dug object" << std::endl;
```

Comment:
`Punched object`

Reply:
I changed the string to "Pressed place button while looking at object". More verbose but less ambiguous